README.md

```diff
@@ -54,7 +54,7 @@ Set<String> apiModules = [
 
 // Add each module as a dependency
 apiModules.forEach {
-    modImplementation(fabricApi.module(it, FABRIC_API_VERSION))
+    include(modImplementation(fabricApi.module(it, FABRIC_API_VERSION)))
```

Comment:
I believe we should mention the fact this includes the module inside the built jar also with that change.

Otherwise yes this is fine

Reply:
I initially thought that it should be so. But the original code didn't work for me at all, so I added include and it worked. Perhaps it was my mistake. Ok, I will describe your request a little later ;)

Reply:
Will my change work for Kotlin (add "include")? I haven't worked with Kotlin, so I don't know. It's just that these two codes should produce the same result.

Reply:
Not exactly. Kotlin's type checking would not exactly like that.

In kotlin `modImplementation` and any other gradle configuration returns a `Dependecy?` which is nullable. This won't work if you chain the `include` like you do in groovy. You'd either need to make a custom configuration and add that configuration to both `modImplementation` and `include` or use the `?.let { include(it) }` syntax after the last bracket on `modImplementation(...)`

Reply:
Oh, I'll fix mistake I made in the new commit now.